app/views/shopping_shared/_tabs.html.haml

```diff
@@ -12,4 +12,7 @@
     - shop_tabs.each do |tab|
       %div{id: "#{tab[:name]}_panel", "data-tabs-and-panels-target": "panel #{'default' if tab[:default]}" } 
         .page-view
-          = render "shopping_shared/tabs/#{tab[:name]}"
+          - if tab[:custom]
+            = render "shopping_shared/tabs/custom", order: @order
```

Comment:
You are passing the order here but then you use `@distributor`. So maybe we don't need this if-branch?

Reply:
Right. Removed: ba399145d2

The `if` branch is necessary since in one case we retrieve the `custom` template in all others cases, we retrieve the `tab[:name]` template (which is not `custom` for the custom tab)

Reply:
Ah, I see. If we had named it `custom` then we wouldn't need the `if`. But the name is showing in the URL and it looks nicer to have the title in there. :+1: